docs/releases/5.0.md

```diff
@@ -73,3 +73,40 @@ The following features deprecated in Wagtail 4.0 have been fully removed. See [W
 ### `Page.get_static_site_paths` method removed
 
 The undocumented `Page.get_static_site_paths` method (which returns a generator of URL paths for use by static site generator packages) has been removed. Packages relying on this functionality should provide their own fallback implementation.
+
+### Progress button (`button-longrunning`) now implemented with Stimulus
+
+The `button-longrunning` class usage has been updated to use the newly adopted Stimulus approach, the previous data attributes will be deprecated in a future release.
+
+If using the old approach, ensure any HTML templates are updated to the new approach before the next major release.
+
+### Old syntax
```

Comment:
The three headings from this one onwards should use the fourth level (`####`)

Reply:
Fixed in 55483b5424b1df707afbf38ca2298dba5095ec00 thanks Sage